routing/index_graph_starter_joints.hpp

```diff
@@ -45,14 +49,23 @@ class IndexGraphStarterJoints
     GetEdgeList(vertex, false /* isOutgoing */, edges);
   }
 
-  WorldGraph::Mode GetMode() const { return m_starter.GetMode(); }
+  WorldGraphMode GetMode() const { return m_graph.GetMode(); }
   // End of A* interface.
 
-  IndexGraphStarter & GetStarter() { return m_starter; }
+  Graph & GetGraph() { return m_graph; }
 
   /// \brief Reconstructs JointSegment by segment after building the route.
   std::vector<Segment> ReconstructJoint(JointSegment const & joint);
 
+  // Не можем проверять с помощью segment.IsRealSegment(), потому что при генерации карты,
+  // m_numMwmId всегда фейковый.
```

Comment:
Translate pls

Reply:
```
// Can not check segment for fake or not with segment.IsRealSegment(), because all segments
  // have got fake m_numMwmId during mwm generation.
```